src/core/EnvelopeAndLfoParameters.cpp

```diff
@@ -94,12 +94,12 @@ EnvelopeAndLfoParameters::EnvelopeAndLfoParameters(
 	Model( _parent ),
 	m_used( false ),
 	m_predelayModel( 0.0, 0.0, 2.0, 0.001, this, tr( "Env pre-delay" ) ),
-	m_attackModel( 0.0, 0.0, 2.0, 0.001, this, tr( "Env attack" ) ),
-	m_holdModel( 0.5, 0.0, 2.0, 0.001, this, tr( "Env hold" ) ),
-	m_decayModel( 0.5, 0.0, 2.0, 0.001, this, tr( "Env decay" ) ),
-	m_sustainModel( 0.5, 0.0, 1.0, 0.001, this, tr( "Env sustain" ) ),
-	m_releaseModel( 0.1, 0.0, 2.0, 0.001, this, tr( "Env release" ) ),
-	m_amountModel( 0.0, -1.0, 1.0, 0.005, this, tr( "Env mod amount" ) ),
+	m_attackModel( 0.025, 0.0, 2.0, 0.001, this, tr( "Env attack" ) ),
+	m_holdModel( 0.0, 0.0, 2.0, 0.001,     this, tr( "Env hold" ) ),
+	m_decayModel( 0.6, 0.0, 2.0, 0.001,    this, tr( "Env decay" ) ),
+	m_sustainModel( 0.0, 0.0, 1.0, 0.001,  this, tr( "Env sustain" ) ),
+	m_releaseModel( 0.2, 0.0, 2.0, 0.001,  this, tr( "Env release" ) ),
+	m_amountModel( 1.0, -1.0, 1.0, 0.005,  this, tr( "Env mod amount" ) ),
 	m_valueForZeroAmount( _value_for_zero_amount ),
```

Comment:
Here is an earlier, similar, fix that had to be reverted: https://github.com/LMMS/lmms/pull/3090

Reply:
The fix only influence new made instruments. Not even instruments in current default template, has the envelope. Non of the existing saved presets should be changed, but i must admit, that i did not test **samples**. I will try and see if they act different from other items in *Instruments*
About default template. How about having a nice zasfx preset, instead of the sine-beeping 3oC? Afaiu even MAC now supports zasfx?

Reply:
> Me said : must admit, that i did not test samples.

and i should have done
You are correct, samples are different from presets. They do not have their envelope 'baked in' They uses current default
That is a terrible descission, but that does not count. It is so. It cant be revoked, that will crush backward compatibility -the holy graal.. 
But i wonder
Several part of the code differentiates between being for MAC / Linux / win
Could we make a similar differentiation between an envelope in *Instruments* and one in AFP -eg **samples**
Then we could have nice default envelopes in instruments, but preserve the no-envelope-state for Samples. I will take a new look at envelope again -eg
`src/core/EnvelopeAndLfoParameters.cpp`
And see if i can make a distinction between Instruments and Samples in AFP